getting_started_materials/intro_tutorials_and_guides/11_Introduction_to_Strings.ipynb

```diff
@@ -0,0 +1,3040 @@
+{
```

Comment:
<p>If we want to show the actual df/series, then we need print out the necessary and pertinent information.  If the information isn't necessary or manipulates the pandas/cudf df/series, then we should remove it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/notebooks-contrib/pull/320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>All pertinent information are printed and or labeled accurately</p>